audio-worklet/src/noisegate-script-processor.js

```diff
@@ -56,19 +56,14 @@ class NoiseGate {
     const bufferSize = options.bufferSize || 0;
     this.attack = options.attack || 0;
     this.release = options.release || 0;
-    
+
     // The time constant of the filter has been set experimentally to balance
     // roughly delay for high frequency suppression.
-    let timeConstant;
-    if (options.timeConstant != null)
-      timeConstant = options.timeConstant;
-    else
-      timeConstant = 0.0025;
+    let timeConstant = (typeof options.timeConstant === 'undefined') ?
+        0.0025 : options.timeConstant;
```

Comment:
Sure, I will do this check as well as the threshold check in a later PR. Probably easiest to make this change the worklet integration PR. What do you think?